app/helpers/import_rooms.rb

```diff
@@ -0,0 +1,30 @@
+require 'csv'
+require 'point'
+require 'room'
+require 'building'
+require 'polyline'
+
+rooms = %w[A B C D E F G H I]
```

Comment:
A comment like `"extend to import more rooms"` would be useful I think

Reply:
gonna add that